app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,60 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, :valid_user, :check_expiration, only: %i(edit update)
+
+  def new; end
+
+  def edit; end
+
+  def update
+    if params[:user][:password].empty?
```

Comment:
```suggestion
    if params[:user][:password].blank?
```
Giúp anh phân biết giữa blank? empty? và nil? nhé

Reply:
nil? trả về true khi đối tượng trả về là nil còn lại là false.
empty? dùng cho collection vs string, nếu object có giá trị nil thì bị raise exception 
blank? nó trả true khi gặp cả 2 trường hợp object có giá trị nil với false, giải quyết được vấn đề exception của empty?